coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/parsetree/ParseTreeBuilder.java

```diff
@@ -235,6 +246,64 @@ private SqlNode visitLateralViewInternal(ASTNode node, ParseContext ctx, boolean
     return joinNode;
   }
 
+  private SqlNode visitLateralViewJsonTuple(List<SqlNode> sqlNodes, List<SqlNode> aliasOperands, SqlCall sqlCall) {
+    /*
+     Represent
+        LATERAL VIEW json_tuple(json, p1, p2) jt AS a, b
+     as
+        LATERAL (
+          SELECT
+            IF(p1 is supported JSON key, get_json_object(json, '$["${p1}"]'), NULL) a,
+            IF(p2 is supported JSON key, get_json_object(json, '$["${p1}"]'), NULL) b
```

Comment:
Could you expand/replace `is supported JSON key` in the comment to follow the code below?

Reply:
originally i had this very precise (actual expression), but thought it doesn't add value, and actually makes the intent less clear